addons/httprox/common/src/main/java/org/commonjava/indy/httprox/util/HttpProxyConstants.java

```diff
@@ -24,6 +24,7 @@
 
 public final class HttpProxyConstants
 {
+    public static final int MITM_SO_TIMEOUT_MINUTES = 30;
 
```

Comment:
Would it be a good idea to make this configurable with that value as a default?

Reply:
Yep. I will add a config entry for it.